classes/submission/reviewAssignment/ReviewAssignmentDAO.inc.php

```diff
@@ -573,11 +573,13 @@ function getLastReviewRoundReviewAssignmentByReviewer($submissionId, $reviewerId
 				(int) $reviewerId
 		);
 
+		$isSqlServer = Config::getVar('database', 'ms_sql');
 		$result = $this->retrieve(
 				$this->_getSelectQuery() .
 				' WHERE	r.submission_id = ? AND
 				r.reviewer_id = ?
-				ORDER BY r2.stage_id DESC, r2.round DESC LIMIT 1',
+				ORDER BY r2.stage_id DESC, r2.round DESC
+				' . ($isSqlServer ? 'OFFSET 0 ROWS FETCH NEXT 1 ROWS ONLY' : 'LIMIT 1'),
```

Comment:
I think you could use `DAO::retrieveRange` for all DB types, and remove this extra consideration for SQL Server.

Reply:
(rather than `DAO::retrieve` as it currently is written)

Reply:
You are right, gonna go this way instead.